web/styles/zulip.css

```diff
@@ -3189,3 +3189,43 @@ select.invite-as {
         padding: 5px;
     }
 }
+
+.tabs-container {
```

Comment:
I think this probably should go in `app_components.css`; we generally put fairly well-defined reusable components there.

Reply:
I moved this while resolving the merge conflicts.